vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,173 @@
+
+- [Task Notes](https://wiki.dendron.so/notes/SEASewZSteDK7ry1AshNG/)
+  - [Contribute to the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)
+- [Note Traits](https://wiki.dendron.so/notes/EQoaBI8A0ZcswKQC3UMpO/)
+- [The Open PKM Catalogue](https://pkm.dendron.so): This work-in-progress is meant to be a reference of all things PKM.
+- Improvements to [importing existing Markdown](https://wiki.dendron.so/notes/f23a6290-2dec-45dc-b616-c218ee53db6b/)
+- Better workflows, such as using [multiple vaults with private and public notes](https://wiki.dendron.so/notes/6DZiBObwhZNYRjnokQ422/)
+
+Though half of respondents said they were developers, the other half of new users came from tech and non-tech backgrounds. This includes many users where Dendron is their introduction to [VS Code](https://code.visualstudio.com/), [git, and GitHub](https://wiki.dendron.so/notes/vsAxYld4zsbD2XjKoh7OP/).
+
+Here are a few areas where we've taken action:
+
+- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/)
+- GitHub Templates for easier, more automated publishing
+  - [Publish with Netlify](https://wiki.dendron.so/notes/yetuum6o9wZi6eVJQBbQb/)
+  - [Publish with GitHub Pages and GitHub Actions](https://wiki.dendron.so/notes/FnK2ws6w1uaS1YzBUY3BR/)
+- Improved `Change Workspace` and `Initialize Workspace` UX
+
+Many of our new users are coming to Dendron as their first tool for personal knowledge management than from any other tool. Even then, the amount of different workflows and tooling that people come from (or are using in parallel) is a massive list.
+
+We have a [migration guide](https://wiki.dendron.so/notes/f9b4fc21-7613-4c8a-9257-cec4c06b67f9/) that can be referenced for people wanting to import their existing notes into Dendron. With so many tools out there, we are open to [any docs contributions and improvements](https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.guides.migration.md) whenever someone transforms into a dendronite.
+
+Within the Dendron Discord, users have mentioned their interest in _interoperability_: they want a single source that they export/import their notes to/from. Our users do this with pods or their own tooling. Different tools co-exist in workflows and can be used concurrently, which further strengthens our belief [that pods are important](https://wiki.dendron.so/notes/66727a39-d0a7-449b-a10d-f6c438185d7f/).
+
+- The Dendron team reflects this. For example, we make use of Airtable every day and will be releasing an Airtable pod in the near future.
+
+Dendron also stands out compared to other static site generator (SSG) tools due to our UX/DX in managing and taking notes. The quarter of respondents expressing no interest in publishing may also reflect the absence of popular SSGs (Sphinx, Hugo, Jekyll, etc.) in response to tools our users are familiar with.
```

Comment:
This is introduced out of the blue. Would not introduce new things in conclusion. Next steps:
- add a part about publishing in the body of the blog
- update this part accordingly